docs/alerts/monitors/create-monitor.md

```diff
@@ -250,6 +233,23 @@ Log monitor triggers are evaluated by balancing the requirement of timely alert
 
 <img src={useBaseUrl('img/monitors/metrics-query.png')} alt="metrics query.png" style={{border: '1px solid gray'}} width="600" />
 
+#### Prerequisites
+
+To fully leverage AI-driven alerts for metrics monitors, you'll need:
+
+* **Automation Service**. Required for linking playbooks to metrics-based monitors.
+* **Metrics data**. Metrics data must be sent to Sumo Logic for anomaly detection.
+* **Metrics aggregation**. Queries that return multiple time series should be aggregated (for example, using `sum` or `avg` operators) before applying anomaly detection.
```

Comment:
This will be required for all queries, not just queries that return multiple time series.

Reply:
thanks! changed to...

* **Metrics aggregation**. Queries should be aggregated (for example, using `sum` or `avg` operators) before applying anomaly detection.